### PR TITLE
Finish milestone journey paragraphs

### PR DESCRIPTION
**Pull request by jasmineortega**

### CODE REVIEW

File: notebooks/Milestone4.ipynb [hunk 1]
```diff
@@ -221,7 +221,15 @@
...
     "rubric={mechanics:10}\n",
-    ">There is no format or structure on how you write this. (also, no minimum number of words).  It's your choice on how well you describe it."
+    ">There is no format or structure on how you write this. (also, no minimum number of words).  It's your choice on how well you describe it.\n",
+    "\n",
+    "In Milestone 1, we learned the pain of dealing with big data on a local machine. We downloaded files using API and combined the data to create one extremely large rainfall dataset of over 62,000,000+ rows. To minimize memory usage while working with this large data set, we loaded the data in chunks of 10,000,000 and converted the datatypes to `float32`. Finally, we conducted a simple EDA using the optimized data set from CSV, which took about a minute to execute. To compare the CSVs versus more efficient data files formats, we timed the data loading and EDA using a parquet file and. For each machine, it took about 4 seconds. It was interesting to see how long each task ran on different machines. For example, while everyone on the team has a MacBook, it was clear that the 16-GB RAM M1 chip was superior in execution time.\n",
```
Comment (lynnwbl) [inline notebooks/Milestone4.ipynb]: To compare the CSVs versus more efficient data files formats, we timed the data loading and EDA using a parquet file<del> and </del>.

File: notebooks/Milestone4.ipynb [hunk 2]
```diff
@@ -221,7 +221,15 @@
...
+    "\n",
+    "In Milestone 2, we created a virtual server on AWS by initializing an EC2 instance. Additionally, we set JupyterHub for the team. This was a relatively straightforward experience, however, we were unable to execute the optional step of setting up a shared JupyterHub notebook environment, which seemed like a useful feature for this milestone. However, it was easy to see the value of setting up a shared notebook environment over everyone sharing the root username and password. In this milestone, we also set up the S3 bucket to host our rainfall dataset extracted from milestone 1 and wrangled the data in preparation for next week's task, \n",
```
Comment (lynnwbl) [inline notebooks/Milestone4.ipynb]: ...and wrangled the data in preparation for next week's task<del>,<del>.

**Review by lynnwbl — CHANGES_REQUESTED** [review]
Just found some small typos. Looks good to me! Thanks!

**Review by jasmineortega — COMMENTED** [review]
Updated!

**Review by lynnwbl — APPROVED** [review]

**Pull request merged**